### PR TITLE
sync_pgviews

### PR DESCRIPTION
**Pull request by scott-w**
I've modified the codebase to use sync_pgviews per #2.

I've also bumped the version number to 0.0.2.

It would be good if you could merge this and put it on pypi, so we can give it a proper thrashing to go into our production system.

Cheers,

Scott,

P.S. Sorry about all the commits. That's what I get for writing code on a Friday afternoon near Christmas :-)

### CODE REVIEW

File: django_postgres/view.py [hunk 1]
```diff
@@ -51,22 +51,26 @@ def realize_deferred_projections(sender, *args, **kwargs):
 models.signals.class_prepared.connect(realize_deferred_projections)
 
 
-def create_views(sender, *args, **kwargs):
+def create_views(models_module, *args, **kwargs):
```
Comment (zacharyvoase) [inline django_postgres/view.py:54]: I'd rather take `sender` as per the Django documentation: https://docs.djangoproject.com/en/dev/topics/signals/#receiver-functions

Comment (zacharyvoase) [inline django_postgres/view.py:54]: EDIT: I realized you're getting rid of this as a signal handler and making it just another function. In that case we should just drop the variadic arguments as well.

**Comment by zacharyvoase** [comment]
OK, I like the work you've done; it all seems pretty straightforward. I'm going to rebase and squash this branch into a couple of concise commits, if you don't mind. Hopefully I'll have it done in the next 24h.

**Comment by scott-w** [comment]
Yeah, go ahead. I would have done it myself, but I haven't spent much time with Git :-)

**Comment by scott-w** [comment]
I've just updated the documentation to reference the `sync_pgviews` command, and how it works with South.

**Comment by zacharyvoase** [comment]
Take a look at this branch I've set up, which incorporates your changes, plus some edge-case handling that I added (namely `--force` and `--update`): https://github.com/zacharyvoase/django-postgres/compare/sync_pgviews

**Comment by zacharyvoase** [comment]
Merged the `sync_pgviews` branch into master.

**Comment by scott-w** [comment]
Awesome. I've put the sync_pgviews branch into our development branch, so we'll be able to put it in a production environment before the summer.

**Pull request closed without merging**